apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/LoggingUtils.java

```diff
@@ -0,0 +1,38 @@
+package gov.cms.bfd.server.war.commons;
+
+import gov.cms.bfd.server.sharedutils.BfdMDC;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** A set of methods for various logging purposes i.e. MDC */
+public class LoggingUtils {
+  private static final Logger LOGGER = LoggerFactory.getLogger(LoggingUtils.class);
+
+  /**
+   * Output list of benefificiary IDs to MDC logging
```

Comment:
Suggestion for rewording:

Log a list of beneficiary IDs to the BfdMDC under the 'bene_id' key.